yt_dlp/extractor/nhk.py

```diff
@@ -539,9 +539,31 @@ class NhkRadiruIE(InfoExtractor):
         },
     }]
 
+    _api_config = None
+
+    def _extract_extended_description(self, episode_id, aa_vinfo2, aa_vinfo3):
+        service, area = aa_vinfo2.split(",")
```

Comment:
I think it would be simpler and safer if we pass the whole `episode` dict to this method and deal with it inside
```suggestion
    def _extract_extended_description(self, episode_id, episode):
        service, area = traverse_obj(episode, ('aa_vinfo2', {str}, {lambda x: (x or '').partition(',')}))
        aa_vinfo3 = traverse_obj(episode, ('aa_vinfo3', {str}))
        if not (service and area and aa_vinfo3):
            return
```
&nbsp;
and then below
```py
        description = self._extract_extended_description(episode_id, episode)
```